README.md

```diff
@@ -1,4 +1,4 @@
-# Welcome to Pathfinder
+# Welcome to  
```

Comment:
Hmm, is this change intentionally part of this PR?

Reply:
THanks and fixed :)